src/development/tools/devtools/performance.md

```diff
@@ -18,28 +18,39 @@ your application. It consists of three parts, each increasing in granularity.
 * Timeline events chart
 * CPU profiler
 
-{{site.alert.note}}
+{{site.alert.secondary}}
   **Use a profile build of your application to analyze performance.**
-  Frame rendering times are not indicative of release performance
+  Frame rendering times aren't indicative of release performance
   unless your application is run in profile mode.
```

Comment:
**suggestion:** Remove the nominalization.
```suggestion
  Don't take frame rendering times to equal release performance
  unless you run your app in profile mode.
```

Reply:
Even in profile mode, performance isn't the same as release mode. This is misleading.